internal/typescript_mock.bzl

```diff
@@ -0,0 +1,12 @@
+def _mock_typescript_lib(ctx):
```

Comment:
give a quick doc, like "allows testing that node_jasmine_test will work with ts_library from rules_typescript without introducing a circular dependency"